examples/tools/cmake/cmake_toolchain/use_different_toolchain_generator.rst

```diff
@@ -0,0 +1,104 @@
+
+.. _examples-tools-use-different-toolchain-generator:
+
+Using CMakeToolchain with Different Generators: Ninja Example
+=============================================================
+
+This guide demonstrates how to use ``CMakeToolchain`` with predefined generators like ``Ninja`` and how to configure it to use different generators.
+
+Creating a Basic Project
+-------------------------
+We will create a basic project based on the ``cmake_exe`` template as an example of a C++ project:
+
+.. code-block:: bash
+
+    $ conan new -d name=foo -d version=0.1.0 cmake_exe
```

Comment:
Positional argument first, please, as we recommend ourselves
```suggestion
    $ conan new cmake_exe -d name=foo -d version=0.1.0
```

Reply:
Ops! I'll update the other reference too: https://docs.conan.io/2/examples/tools/cmake/cmake_toolchain/build_project_cmake_presets.html#cmaketoolchain-building-your-project-using-cmakepresets

Reply:
Done, fixed all usages for `cmake new `